docs/source/using/addons.adoc

```diff
@@ -109,7 +109,8 @@ This can be useful in cases where you know that a command such as `oc` might tak
 
 [NOTE]
 ====
-Trying to use an undefined command will cause an error when the add-on gets parsed.
+- Trying to use an undefined command will cause an error when the add-on gets parsed.
+- Exit status of a Command start with `!` is ignored.
```

Comment:
@praveenkumar I didn't get this line. Could you elaborate?

May be it should be written as
```
- Failure of command starting with `!` is ignored 
```

Reply:
Below section I feel should come after `Built-in Variables` section.
```
=== Special characters

Add-on file contains some special character that has special meaning. 
Below are few special characters which one can use to have special behavior:
- `#` : This character is reserved for comment character.
- `!` : When used at the beginning of any command, the failure of the command will be ignored and will not exit the addon execution
```
Somewhere on the similar lines.

@robin-owen could you help here to improve it?

Reply:
I think a section is certainly appropriate, but I think placing it directly before Variable Interpolation makes the most sense. The special characters we're talking about here are quite relevant to the available commands, and having them documented in between documentation about variables feels out of place to me.

As for the wording itself, it could use some improvement. Once we have the placement worked out, I can send a PR to praveenkumar:issue_1130 to assist with the writing. :)

Reply:
@budhrg @robin-owen problem is `#` is also used as part of metadata definition for so might be we need rewording around it, Meanwhile, I added `- Execution failure of command starting with `!` is ignored` which at this moment feel more appropriate to me.

Reply:
As I said documentation for `!` is just a line and easily not visible :)

Reply:
WDYT @LalatenduMohanty ?